cms/static/cms/js/modules/cms.modal.js

```diff
@@ -666,6 +672,32 @@
 
                 // update title
                 this.ui.titlePrefix.text(el.text());
+            },
+
+            /**
+             * _deletePlugin removes a plugin once created when clicking
+             * on delete or the close item. If we don't do this, an empty
+             * placeholder is generated
+             * https://github.com/divio/django-cms/pull/4381 will eventually
+             * provide a better solution
+             *
+             * @param hideAfter Boolean
```

Comment:
I think it might be a bit overkill to pass an object just yet, as there is currently only one exception. If we add more than it would make sense?